app/controllers/sections_controller.rb

```diff
@@ -8,23 +8,42 @@ def index
     redirect_to courses_path
   end
 
+  def new
+    @course = Course.find(params[:course_id])
+  end
+
+  def create
+    @course = Course.find(params[:course_id])
+    @section = Section.new(section_params)
+    if @section.save
+      redirect_to course_path(@course), notice: 'Section saved.'
```

Comment:
how about `redirect_to course_path(@section.course)`, and remove line 17? Keeps with the "1 ivar per controller action" Sandi Metz rule of thumb.